src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3869,6 +3869,9 @@
   <data name="Argument_EmptyString" xml:space="preserve">
     <value>The value cannot be an empty string.</value>
   </data>
+  <data name="Argument_EmptyOrWhiteSpaceString" xml:space="preserve">
+    <value>The value cannot be an empty string or composed entirely of whitespace.</value>
```

Comment:
any value in including the value here, so in a log one can tell whether it's whitespace or empty? (or have separate messages for empty and whitespace)

Reply:
I don't think so.